packages/discord.js/typings/index.d.ts

```diff
@@ -6221,6 +6248,18 @@ export interface IntegrationAccount {
 
 export type IntegrationType = 'twitch' | 'youtube' | 'discord' | 'guild_subscription';
 
+export type IntegrationTypesConfigurationParameters = {
+  [T in keyof ClientApplicationInstallParams]: ClientApplicationInstallParams[T];
+};
```

Comment:
Seems this is equivalent to
```ts
export type IntegrationTypesConfigurationParameters = ClientApplicationInstallParams
```

 We can probably remove one of these and rename the other to a more fitting name for both (for documentation too)